### ISSUE TITLE
Regular Expressions

### ISSUE THREAD

**Issue by tvooo** [CodeYourFuture/syllabus-archive#28]
Hey folks,

I just did a quick search for Regexp in the repo and nothing turned up. I'd like to discuss if and when we should introduce them. Sure, it's a quite advanced topic, but I believe it is somewhat necessary to know about - at least some basics.

Any thoughts?

**Comment by rarmatei** [comment]
You're right, if they start work full-time after, they're bound to run into them eventually, and it's better if they've at least seen them before. 

Should we save them for Lesson 4 (#12)? If you go to the lesson's issue, you'll find I already suggested some topics we can cover, in order of what I feel is more important. Feel free to add to that, and we can start prioritising topics there?
